syntax = "proto3";

package Packet;

message MurdererPacket {
}